corehq/apps/reports/filters/users.py

```diff
@@ -410,3 +429,75 @@ def get_user_toggle(request):
     elif group or individual:
         show_filter = False
     return toggle, show_filter
+
+
+class SubmitHistoryFilter(ExpandedMobileWorkerFilter):
+    options_url = 'emwf_options_all_users'
+
+    @property
+    @memoized
+    def utils(self):
+        return SubmitHistoryUtils(self.domain)
+
+    @property
+    def filter_context(self):
+        context = super(SubmitHistoryFilter, self).filter_context
+        url = reverse(self.options_url, args=[self.domain])
+        context.update({'endpoint': url})
+        if self.request.project.uses_locations:
+            context.update({'search_help_inline': self.search_help_inline})
+        return context
```

Comment:
This method can be deleted, it's a copy of the parent class's

Reply:
https://github.com/dimagi/commcare-hq/pull/21564/commits/ce0c877d1c827f815143d59c38e2f7a990e5a514